packages/sdks/snippets/angular/src/app/app.module.ts

```diff
@@ -12,10 +12,11 @@ import { BlogArticleComponent } from './blog-article/blog-article.component';
 import { CatchAllComponent } from './catch-all/catch-all.component';
 import { CustomChildComponent } from './custom-child/custom-child.component';
 import { CustomHeroComponent } from './custom-child/custom-hero/custom-hero.component';
+import { CustomColumnsComponent } from './editable-regions/custom-columns/custom-columns.component';
+import EditableRegionComponent from './editable-regions/editable-regions.component';
```

Comment:
can we use named export for `EditableRegionComponent` to maintain consistency?

Reply:
This is an overlook. Made the change and pushed the code.